### ISSUE TITLE
[Texteditor]: WCAG and other issues

### ISSUE THREAD

**Issue by ivarkallejarv** [TEDI-Design-System/react#107]
## Problem

- Use TextEditor component
- TextEditor text input is missing accessible name
- Formatting buttons (bold/italic/bullet) are missing accesible texts

![Image](https://github.com/user-attachments/assets/226468c2-6640-4c45-9792-181d5ea5db07)
